impeller/entity/contents/solid_color_contents.cc

```diff
@@ -60,7 +63,8 @@ bool SolidColorContents::Render(const ContentContext& renderer,
       };
   return ColorSourceContents::DrawGeometry<VS>(
       renderer, entity, pass, pipeline_callback, frame_info,
-      [](RenderPass& pass) {
+      [&frag_info, &host_buffer](RenderPass& pass) {
+        FS::BindFragInfo(pass, host_buffer.EmplaceUniform(frag_info));
```

Comment:
Unrelated to this PR, but "EmplaceUniform" is a poor name if it isn't taking in an rvalue.